webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -65,11 +69,22 @@ const ContentViewFilters = ({ cvId }) => {
         },
       ];
 
-      newRows.push({ cells });
+      newRows.push({ cells, id });
     });
     return newRows;
   };
 
+  const bulkRemove = () => {
+    const filterIds = rows.filter(({ selected }) => selected).map(({ id }) => id);
+    dispatch(deleteContentViewFilters(cvId, filterIds, () =>
```

Comment:
@jeremylenz What do you think of this syntax for the callback? 
The dispatch is in the component as I remember you requested, I think this looks clean.

Reply:
I hate that the next dispatch is nested inside a callback! I totally understand why it's that way, though. I'll keep trying to think if there's any better way..

Reply:
Just to remind you that whatever you elders come up with here will probably be the pattern kids follow for other deletion flows on the CV UI since this is the first DELETE API call so far..No pressure.. 😛

Reply:
:older_man:

Reply:
The yays have it! Resolving convo for now!